doc/reference/configuration/index.rst

```diff
@@ -28,40 +28,100 @@ Command options
 
 .. _index-tarantool_version:
 
-..  option:: -V, --version
+..  option:: -v, -V, --version
 
-    Print product name and version, for example:
+    Print the product name and version.
+
+    **Example**
 
     ..  code-block:: console
 
-        $ ./tarantool --version
-        Tarantool 2.10.4-0-g816000e
-        Target: Darwin-x86_64-Release
+        % tarantool --version
+        Tarantool 2.11.1-0-g96877bd
+        Target: Darwin-arm64-Release
         ...
 
     In this example:
 
-    “Tarantool” is the name of the reusable asynchronous networking
-    programming framework.
+    *   ``2.11.1`` is a Tarantool version.
+        Tarantool follows semantic versioning, which is described in the :ref:`Tarantool release policy <release-policy>` section.
+
+    *   ``Target`` is the platform Tarantool is built on.
+        Platform-specific details may follow this line.
+
+
+..  option:: -e EXPR
+
+    Execute the 'EXPR' string. See also: `lua man page <https://www.lua.org/manual/5.3/lua.html>`_.
```

Comment:
I think that a link to Lua 5.1 manual would be more relevant for us, if we want to highlight similarity of some options to PUC-Rio Lua ones.